pkg/tnf/handlers/handler_template/handler_test.tmpl

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2020-2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package {{ .Handlername }}
+
+import (
+	"time"
+)
+
+const (
+	testTimeoutDuration = time.Second * 2
+)
+
+//TODO:
+func {{ .Handlername }}_Args{
```

Comment:
// {{ .Handlername }}_Args() 
 func Test{{ .Handlername }}_Args(){
// Todo : write test for function {{ .Handlername }}_Args()
}